### PR TITLE
Text blocks

### PR DESCRIPTION
**Pull request by aloncohenn**

### CODE REVIEW

File: src/plugins/blocksmith/components/blocksmith/index.jsx [hunk 1]
```diff
@@ -347,6 +347,69 @@ class Blocksmith extends React.Component {
     updateBlocks.clearCache();
   }
 
+  /**
+   * Given a canvas block, creates a DraftJs block (with data) for said block at the active block.
+   *
+   * @param {*} canvasBlock                - a triniti canvas block
+   * @param {boolean} shouldSelectAndStyle - whether or not to select and style the new block
+   */
+  handleAddCanvasBlock(canvasBlock, shouldSelectAndStyle = false) {
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/blocksmith/index.jsx:356]: @aloncohenn i get a lint error if you move this fn, do you?

Screen Shot 2021-02-18 at 8 42 42 AM

File: src/plugins/blocksmith/utils/isAdvancedBlockSelected.js [hunk 2]
```diff
@@ -12,20 +12,5 @@ import getSelectedBlocksList from './getSelectedBlocksList';
  */
 export default (editorState) => {
```
Comment (efjacobson) [inline src/plugins/blocksmith/utils/isAdvancedBlockSelected.js:13]: @aloncohenn since this is simpler now i would rename the util to isAtomicBlockSelected

**Review by efjacobson — CHANGES_REQUESTED** [review]

File: src/plugins/blocksmith/components/blocksmith/index.jsx (hunk 1 above)
Comment (aloncohenn) [reply src/plugins/blocksmith/components/blocksmith/index.jsx]: i'm getting a lint error when i move it back to where it was, i moved again

File: src/plugins/blocksmith/utils/isAdvancedBlockSelected.js (hunk 2 above)
Comment (aloncohenn) [reply src/plugins/blocksmith/utils/isAdvancedBlockSelected.js]: @efjacobson renamed

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 
 ## vN.N.N
+* Make only atomic blocks be advanced blocks
```
Comment (efjacobson) [inline CHANGELOG.md:5]: @aloncohenn more description of why we are doing this, like the ticket name, would be good

**Review by efjacobson — APPROVED** [review]
approved but please make changelog adjustment from comment

**Pull request merged**